Makefile

```diff
@@ -9,17 +8,20 @@ test_files =$(wildcard tests/**/*.ts)
 test_out_files = $(test_files:tests/%.ts=tests/%.lua)
 test_out_init = $(test_init:tests/%.ts=tests/%.lua)
 
-compile:
+install:
+	npm ci
+
+compile: install
```

Comment:
**issue**: I'm not sure if it's a good idea to make `compile` call `install` since this target is called multiple times during development.

**suggestion**: create a new target that calls `install` and `compile` and keep `compile` without the call to `install`